modules/nf-core/svtk/rdtest2vcf/meta.yml

```diff
@@ -0,0 +1,49 @@
+name: "svtk_rdtest2vcf"
+description: Convert an RdTest-formatted bed to the standard VCF format.
+keywords:
+  - sort
+tools:
+  - "svtk":
+      description: "Utilities for consolidating, filtering, resolving, and annotating structural variants."
+      homepage: "https://github.com/broadinstitute/gatk-sv/tree/master/src/svtk"
+      documentation: "https://github.com/broadinstitute/gatk-sv/tree/master/src/svtk"
+      tool_dev_url: "https://github.com/broadinstitute/gatk-sv/tree/master/src/svtk"
+      doi: ""
```

Comment:
For DOi see bottom of this page: https://gatk.broadinstitute.org/hc/en-us/articles/9022653744283-GATK-Best-Practices-for-Structural-Variation-Discovery-on-Single-Samples